plf-tomcat-resources/src/main/resources/bin/setenv.bat

```diff
@@ -58,6 +58,7 @@ REM # --------------------------------------------------------------------------
 IF NOT DEFINED EXO_JVM_SIZE_MAX SET EXO_JVM_SIZE_MAX=3g
 IF NOT DEFINED EXO_JVM_SIZE_MIN SET EXO_JVM_SIZE_MIN=512m
 IF NOT DEFINED EXO_JVM_PERMSIZE_MAX SET EXO_JVM_PERMSIZE_MAX=256m
+IF NOT DEFINED EXO_JVM_METASPACE_SIZE_MAX SET EXO_JVM_METASPACE_SIZE_MAX=256m
```

Comment:
It was a bad copy/paste.
We need to find the right value as it's a value related to the native memory.

Maybe we need to align it according to the prerequisites for installing PLF on a server in order to have a consistent value (percentage of native memory?) /cc @vsellier